event/event.go

```diff
@@ -128,7 +130,9 @@ func (e Event) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 	enc.AddString("cloudEventsVersion", e.CloudEventsVersion)
 	enc.AddString("source", e.Source)
 	enc.AddString("eventID", e.EventID)
-	enc.AddString("eventTime", e.EventTime.String())
+	if e.EventTime != nil {
+		enc.AddString("eventTime", e.EventTime.String())
```

Comment:
Should we just make the `Event` struct `EventTime` into a string to begin with, then to `time.Now().String()` to populate?

Reply:
I prefer time.Time because.. it's a time :) First of all it's good pattern to use types where they are meaningful, secondly it enabled all time operations on this data. Right now it's not used but eventually it will be. Also if someone uses our Event struct (it's based on CloudEvents spec) it's better to provide meaningful types.